Prevent incorrect connectInject config (#282)
hashicorp/consul-helm: CHANGELOG.md

## Unreleased

BREAKING CHANGES
  * `client.grpc` defaults to `true` now instead of `false`. This is to make it

>      harder to misconfigure connect [[GH-282](https://github.com/hashicorp/consul-helm/pull/282)]

Reviewer: ```suggestion
     harder to misconfigure Connect [[GH-282](https://github.com/hashicorp/consul-helm/pull/282)]
```

>      If you wish to not have client grpc ports exposed, set `client.grpc` to

Reviewer: ```suggestion
     If you do not wish to enable gRPC for clients, set `client.grpc` to
```
